buildSrc/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -9,22 +9,26 @@ internal class YamlWriter {
       check(base.serviceAccountCredentials.isPresent) { "ServiceAccountCredentials in fladle extension not set. https://github.com/runningcode/fladle#serviceaccountcredentials" }
     }
     check(base.debugApk.isPresent) { "debugApk must be specified" }
-    check(base.instrumentationApk.isPresent xor !base.roboScript.isNullOrBlank()) {
-      """
+    if (base.sanityRobo.get() == false) {
+      check(base.instrumentationApk.isPresent xor !base.roboScript.isNullOrBlank()) {
```

Comment:
On second though. I don't think that allowing `sanityRobo` to override other configuraiton options is a sustainable way to develop this plugin and could lead to confusing results.
Imagine if `roboScript` were to override the `instrumenationApk` property instead of disallowing this configuraiton. We would have a seriously confusing mess of `if` statements.

To start, can we disallow using `sanityRobo` and `instrumentationApk` and / or `roboScript` within the same fladle config?

Reply:
We should keep this behavior for sanityRobo:
> "Sanity robo should be ONLY launched when there is NO, robo-script, test, additional-app-test-apks. This reflects gcloud's behavior"
I copied that from here https://github.com/Flank/flank/pull/1108